.github/workflows/release-artifacts.yaml

```diff
@@ -34,5 +33,5 @@ jobs:
         uses: docker/build-push-action@v2
         with:
           push: true
-          tags: ${{ steps.meta.outputs.tags }}
+          tags: ${{ github.event.release.tag_name }}
```

Comment:
See: https://stackoverflow.com/questions/59518658/how-to-use-github-release-version-number-in-github-action/59521156#59521156

Reply:
This way, we will not have the `latest` and `commit_sha` tags. Could this break something, somewhere? Just making sure that we have checked and we are sure 🙂

Reply:
We don't have the latest anyways, so I think this is fine.